.buildkite/pipelines/flaky_tests/runner.js

```diff
@@ -183,6 +183,18 @@ for (const testSuite of testSuites) {
         concurrency_group: UUID,
         concurrency_method: 'eager',
       });
+    case 'cypress':
+      const cypressSuites = ['not_used', 'security_solution', 'osquery_cypress', 'fleet_cypress'];
+      steps.push({
+        command: `.buildkite/scripts/steps/functional/${cypressSuites[CI_GROUP]}.sh`,
+        label: 'Default Cypress',
```

Comment:
I think it's be great to:
1. validate that the CI_GROUP is set to one of the `keys` from `groups.json` and
2. use the `name` from that group here
If the CI_GROUP is not in the `groups.json` file then we shouldn't try to execute that file, we should just throw an error.